### DISCUSSION TITLE
Text is invisible

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#1783] [Q&A] [answered]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**. Not a question or feature request.
- [X] The issue is **not** already reported at [Issues](https://github.com/gsantner/markor/issues?q=) or [discussions](https://github.com/gsantner/markor/discussions?discussions_q=). _(I've searched it)_.
- [x] Markor **is** up to date. See [Releases](https://github.com/gsantner/markor/tags) for the latest version. Updates are available from [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/) and GitHub.

### Description

The text is invisible, or same color as bg.

![Screenshot_20220723-120337_Markor](https://user-images.githubusercontent.com/45273247/180619469-cf8f5fbd-525a-460d-82bf-1bc5cddcb25f.png)

![Screenshot_20220723-120326_Markor](https://user-images.githubusercontent.com/45273247/180619463-8a507e89-ae99-4406-9c34-113e0fb594c0.png)

### Steps to reproduce

1. Use markor to make a new document 
2. Close markor
3. Open markor/document 

### Information

Android version: lineage_sunfish-userdebug 11 RQ3A.211001.001 4d8fec2965
Device: pixel 4a
App Version: pasting sc because can't copy
![Screenshot_20220723-120830_Markor](https://user-images.githubusercontent.com/45273247/180619586-5c93cdc5-bdf4-473a-a07f-62815b549a22.png)

### Source

F-Droid

### Format / File type

No

### Additional info / Log

```shell
-
```

**Comment by gsantner** [comment, marked as answer]
go to settings -> general -> color scheme, and select one of the presets. should show a working pair of colors then.

> pasting sc because can't copy  

When you click there, the text is copied to clipboard :-).

**Comment by ghost** [comment]
Oh yup, there was a theme option in settings. Switching it from "auto" fixed it. Thanks!

**Reply by gsantner** [reply]
I converted your issue to a discussion, as it rather fits Q&A then bugs.
